projects/output-and-error-handling/server/client.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	resp, err := http.Get("http://localhost:8080") 
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error making HTTP request: %v\n", err)
+		os.Exit(1)
+	}
+	defer resp.Body.Close()
+	//checking conditions
+	switch resp.StatusCode {
+	case http.StatusOK:
+		fmt.Println("Weather Information:")
+		fmt.Println("Response Status:", resp.Status)
+	case http.StatusTooManyRequests:
+		fmt.Fprintf(os.Stderr, "Server too busy. Will retry after 5 seconds\n")
```

Comment:
The problem statement of the project describes that sometimes the server has told us a specific amount of time to wait, rather than always 5 seconds - can you try to handle that case?